CHANGES

```diff
@@ -41,6 +41,8 @@ Tools
   COQFLAGS is now entirely separate from COQLIBS, so in custom Makefiles
   $(COQFLAGS) should be replaced by $(COQFLAGS) $(COQLIBS).
 
+- Remove gallina.
```

Comment:
You were talking about confusion, you should definitely expand this CHANGES entry to avoid some.

Reply:
Good catch.